.github/workflows/cd.yml

```diff
@@ -260,10 +261,9 @@ jobs:
       with:
         prerelease: ${{ contains(github.ref, 'alpha') || contains(github.ref, 'beta') || contains(github.ref, 'rc') }}
         files: ./*
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUBTOKEN }}
```

Comment:
This setting never worked because it used a wrong secret name, thus it was equivalent to setting an empty env `GITHUB_TOKEN`.

The correct name is `secrets.GITHUB_TOKEN`, see https://docs.github.com/en/actions/learn-github-actions/contexts#secrets-context.